lab1.js

```diff
@@ -134,7 +146,16 @@ var tooHungryDay;
 */
 
 // your code goes here
-
+for (var i = 0, j = 1, mealsTotal = 0, averageMeals = 0;i <= mealsPerDay.length, j <= mealsPerDay.length;i++, j++) {
+  mealsTotal += mealsPerDay[i];
+  averageMeals = mealsTotal / j;
+  console.log(averageMeals);
+  if (averageMeals < 4) {
+    if (tooHungryDay === 0) {
```

Comment:
good way to check if you've already assigned a value to tooHungryDay. You can also use a break to break the loop once you've assigned it.